src/utils/dateHelpers.js

```diff
@@ -212,16 +212,16 @@ export default /** @lends module:core.utils.dateHelpers */ {
         }
     },
 
-    getDisplayDate: function (val) {
-        var lang = LocalizationService.langCode,
-            format = dateTimeFormats[lang].condensedDate.general;
+    getDisplayDate: function (val, format) {

```

Comment:
These helpers don't make sense if you pass in the format parameter. It's better to leave them as they were. Instead, use the helpers only if no format were provided. Otherwise, use moment() directly, without helpers.